packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -7,6 +7,13 @@ const propTypes = {
    * The source URL of the content to load.
    */
   src: PropTypes.string.isRequired,
+  /**
+   * Scrolling is default to `true` and allows content inside of the iframe to be scrolled if needed.
+   * When set to `false`, scrolling is disabled. Regardless of the option, there is a logic to resize
+   * the content within the iframe (within [xfc](https://github.com/cerner/xfc#iframe-resizing-config) library)
+   * to be fully visible so that scrolling is not needed.
+   */
+  scrolling: PropTypes.bool,
```

Comment:
Can we rename this prop to be more consistent with Terra conventions for boolean props? Something like `isScrollable` comes to mind, similar to in `terra-overlay`.

https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/overlay/about

Reply:
Good idea. I renamed it and updated the test here.
https://github.com/cerner/terra-framework/pull/1783/commits/e6bc615fb5c779806c1c36ce35f0a3e0ac5a56d1